### ISSUE TITLE
[Bug]: Creating custom driver throws `Class "ValetDriver" not found`

### ISSUE THREAD

**Issue by aaronbushnell** [beyondcode/herd-community#81]
### Operating system version

macOS Ventura 13.5

### System architecture

Intel (x86)

### Herd Version

1.1.1 (Build: 12)

### PHP Version

PHP 8.2.8 (cli) (built: Jul 28 2023 14:23:03) (NTS)

### Bug description

(This occurs in all PHP versions, FYI)

When I place a custom driver in the Valet path it doesn't seem that the driver is ever called.
```sh
~/.config/valet/Drivers
```

However, if I place it in the Herd path (from #17):
```sh
~/Library/Application Support/Herd/config/valet/Drivers
```

I receive the following error:
```
Fatal error: Uncaught Error: Class "ValetDriver" not found in /Users/aaron/Library/Application Support/Herd/config/valet/Drivers/LegacyProjectValetDriver.php:3
```

## Driver code (LegacyProjectValetDriver.php)
```php
<?php

class LegacyProjectValetDriver extends ValetDriver
{
    /**
     * Determine if the driver serves the request.
     *
     * @param  string  z$sitePath
     * @param  string  $siteName
     * @param  string  $uri
     * @return bool
     */
    public function serves($sitePath, $siteName, $uri)
    {
        if (file_exists($sitePath.'/../app') && file_exists($sitePath.'/../dist')) {
            return true;
        }

        return false;
    }

    /**
     * Determine if the incoming request is for a static file.
     *
     * @param  string  $sitePath
     * @param  string  $siteName
     * @param  string  $uri
     * @return string|false
     */
    public function isStaticFile($sitePath, $siteName, $uri)
    {
        if (file_exists($staticFilePath = $sitePath.'/'.$uri)) {
            return $staticFilePath;
        }

        return false;
    }

    /**
     * Get the fully resolved path to the application's front controller.
     *
     * @param  string  $sitePath
     * @param  string  $siteName
     * @param  string  $uri
     * @return string
     */
    public function frontControllerPath($sitePath, $siteName, $uri)
    {
        return $sitePath.'/index.php';
    }
}
```

### Steps to reproduce

Place the example driver code into the Herd driver location and view a site that Herd manages
```
~/Library/Application Support/Herd/config/valet/Drivers
```

### Relevant log output

```shell
Fatal error: Uncaught Error: Class "ValetDriver" not found in /Users/aaron/Library/Application Support/Herd/config/valet/Drivers/LegacyProjectValetDriver.php:3 Stack trace: #0 /Applications/Herd.app/Contents/Resources/valet/cli/Valet/Drivers/ValetDriver.php(90): require_once() #1 /Applications/Herd.app/Contents/Resources/valet/cli/Valet/Drivers/ValetDriver.php(115): Valet\Drivers\ValetDriver::driversIn('/Users/aaron...') #2 /Applications/Herd.app/Contents/Resources/valet/cli/Valet/Drivers/ValetDriver.php(42): Valet\Drivers\ValetDriver::customDrivers() #3 /Applications/Herd.app/Contents/Resources/valet/server.php(54): Valet\Drivers\ValetDriver::assign('/Users/aaron...', 'myproject', '/') #4 {main} thrown in /Users/aaron/Library/Application Support/Herd/config/valet/Drivers/LegacyProjectValetDriver.php on line 3
```

**Comment by aaronbushnell** [comment]
Ah, thanks @mpociot! I'll hang tight and try it again after Herd is migrated. 🙌

**Comment by mpociot** [comment]
This has been fixed with Herd 1.2
